src/main/java/com/mraof/minestuck/block/MSBlocks.java

```diff
@@ -19,681 +19,518 @@
-	public static final Block BLACK_CHESS_BRICKS = getNull();
-	public static final Block DARK_GRAY_CHESS_BRICKS = getNull();
-	public static final Block LIGHT_GRAY_CHESS_BRICKS = getNull();
-	public static final Block WHITE_CHESS_BRICKS = getNull();
-	public static final Block BLACK_CHESS_BRICK_SMOOTH = getNull();
-	public static final Block DARK_GRAY_CHESS_BRICK_SMOOTH = getNull();
-	public static final Block LIGHT_GRAY_CHESS_BRICK_SMOOTH = getNull();
-	public static final Block WHITE_CHESS_BRICK_SMOOTH = getNull();
-	public static final Block BLACK_CHESS_BRICK_TRIM = getNull();
-	public static final Block DARK_GRAY_CHESS_BRICK_TRIM = getNull();
-	public static final Block LIGHT_GRAY_CHESS_BRICK_TRIM = getNull();
-	public static final Block WHITE_CHESS_BRICK_TRIM = getNull();
-	public static final Block CHECKERED_STAINED_GLASS = getNull();
-	public static final Block BLACK_CROWN_STAINED_GLASS = getNull();
-	public static final Block BLACK_PAWN_STAINED_GLASS = getNull();
-	public static final Block WHITE_CROWN_STAINED_GLASS = getNull();
-	public static final Block WHITE_PAWN_STAINED_GLASS = getNull();
+	public static final RegistryObject<Block> BLACK_CHESS_DIRT = REGISTER.register("black_chess_dirt", () -> new Block(Block.Properties.of(Material.DIRT, MaterialColor.COLOR_BLACK).strength(0.5F).sound(SoundType.GRAVEL)));
+	public static final RegistryObject<Block> WHITE_CHESS_DIRT = REGISTER.register("white_chess_dirt", () -> new Block(Block.Properties.of(Material.DIRT, MaterialColor.SNOW).strength(0.5F).sound(SoundType.GRAVEL)));
+	public static final RegistryObject<Block> DARK_GRAY_CHESS_DIRT = REGISTER.register("dark_gray_chess_dirt", () -> new Block(Block.Properties.of(Material.DIRT, MaterialColor.COLOR_GRAY).strength(0.5F).sound(SoundType.GRAVEL)));
+	public static final RegistryObject<Block> LIGHT_GRAY_CHESS_DIRT = REGISTER.register("light_gray_chess_dirt", () -> new Block(Block.Properties.of(Material.DIRT, MaterialColor.COLOR_LIGHT_GRAY).strength(0.5F).sound(SoundType.GRAVEL)));
+	public static final RegistryObject<Block> SKAIA_PORTAL = REGISTER.register("skaia_portal", () -> new SkaiaPortalBlock(Block.Properties.of(Material.PORTAL, MaterialColor.COLOR_CYAN).noCollission().lightLevel(state -> 11).strength(-1.0F, 3600000.0F).noDrops()));
+	static BlockBehaviour.Properties blackChessBricks = Block.Properties.of(Material.STONE, MaterialColor.TERRACOTTA_BLACK).requiresCorrectToolForDrops().strength(1.5F, 6.0F);
+	static BlockBehaviour.Properties darkGrayChessBricks = Block.Properties.of(Material.STONE, MaterialColor.TERRACOTTA_GRAY).requiresCorrectToolForDrops().strength(1.5F, 6.0F);
+	static BlockBehaviour.Properties lightGrayChessBricks = Block.Properties.of(Material.STONE, MaterialColor.TERRACOTTA_LIGHT_GRAY).requiresCorrectToolForDrops().strength(1.5F, 6.0F);
+	static BlockBehaviour.Properties whiteChessBricks = Block.Properties.of(Material.STONE, MaterialColor.TERRACOTTA_WHITE).requiresCorrectToolForDrops().strength(1.5F, 6.0F);
```

Comment:
I'd suggest cleaning these up, renaming them, making them private static final, and potentially writing some of these as helper functions instead. This applies to pretty much anything here that's not a registry object or the deferred register.
I'd also like you to be adding empty newlines to split things up more.

Reply:
Helper function for streamlining which portion? Working on the rest

Reply:
I was thinking of a helper function that creates and returns some block property, as opposed to creating a block property statically. I don't think I had any specific in mind.
Though now that I've given it some more thought, I realize that that because block properties are mutable, there's a point in making all these common properties provided by helper functions.